nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-pubsub-processors/src/main/java/org/apache/nifi/processors/kafka/pubsub/AbstractKafkaProcessor.java

```diff
@@ -141,6 +166,10 @@
         SHARED_DESCRIPTORS.add(CLIENT_ID);
         SHARED_DESCRIPTORS.add(SECURITY_PROTOCOL);
         SHARED_DESCRIPTORS.add(KERBEROS_PRINCIPLE);
+        SHARED_DESCRIPTORS.add(SSL_KEY_PASSWORD);
```

Comment:
@olegz I do agree that it would be nice to allow users to explicitly mark dynamic properties as sensitive. However, even with that capability, providing the option of using an SSLContextService is still preferable, I believe, as it is typically much easier to configure, since you can configure the service once and then use anywhere. I have attached a patch to NIFI-2423 that incorporates the SSL Context Service. Can you please review & if good either merge into your PR or give it a +1 on the JIRA? Then I should be able to merge in this PR.